### PR TITLE
Dice parsing

### PR DESCRIPTION
**Pull request by RichTeaMan**
# Description

Purrito new understands multiple dice, suc as !roll 20d20

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Added some Jest tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,12 +25,15 @@
     },
     "homepage": "https://github.com/djaustin/purrito-bot#readme",
     "dependencies": {
-        "@types/node": "^13.9.1",
```
Comment (djaustin) [inline package.json]: Nice

File: src/__tests__/DiceMaiden/ParseDiceMaiden.test.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+test('Dice rolls in mixed sets are correctly parsed when username has square braces', () => {
+
+    const message = 'example_player[420] Roll: [20, 18], [11] Result: 31'
+
+    const result = parseDiceMaidenMessage(message)
+
+    expect(result.parseSuccessful).toBeTruthy()
+    expect(result.diceRolls).toStrictEqual([20, 18, 11])
+})
```
Comment (djaustin) [inline src/__tests__/DiceMaiden/ParseDiceMaiden.test.ts]: very nice

**Comment by RichTeaMan** [comment]
> 
> 
> Is there any particular reason behind having tests in `src/__tests__/` rather than just in `test/`?

None at all except for the example I was following.

**Comment by RichTeaMan** [comment]
I don't no idea why the checks are failing.

**Comment by djaustin** [comment]
> > Is there any particular reason behind having tests in `src/__tests__/` rather than just in `test/`?
> 
> None at all except for the example I was following.

Fair enough. Does. the build command drop the test files into the dist directory as well? if we move the tests to `test/` we can easily separate them out.

**Comment by djaustin** [comment]
Dew it

**Pull request merged**